zuul.d/projects.yaml

```diff
@@ -3,7 +3,17 @@
     name: packit-service/packit-service-zuul
     check:
       jobs:
-        - linters
+        - packit-service-deployment
     gate:
       jobs:
-        - linters
+        - packit-service-deployment
+
+- job:
```

Comment:
it seems like this job could use a semaphore to prevent parallel execution. Add above:

```
- semaphore:
    name: packit-deployment-test
    max: 1
```

Then in the job, add `semaphore: packit-deployment-test` attribute.